server/graphql/core/src/loader/campaign.rs

```diff
@@ -0,0 +1,28 @@
+use async_graphql::dataloader::*;
+use async_graphql::*;
+use repository::{
+    campaign_row::{CampaignRow, CampaignRowRepository},
+    RepositoryError, StorageConnectionManager,
+};
+use std::collections::HashMap;
+
+pub struct CampaignByIdLoader {
+    pub connection_manager: StorageConnectionManager,
+}
+
+impl Loader<String> for CampaignByIdLoader {
```

Comment:
After some discussions and additional input, we've decided to add a `CampaignByIdLoader`, which can be used as needed (currently only used in Stock Line).

That said, while implementing it, I just couldn't visualise where we would render this information. If we decide it's unnecessary, I'm happy to remove it.